src/main/java/com/minecolonies/coremod/colony/workorders/AbstractWorkOrder.java

```diff
@@ -285,9 +285,13 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
     /**
      * Is this WorkOrder still valid?  If not, it will be deleted.
      *
+     * Suppressing Sonar Rule squid:S1172
+     * This rule does " Unused method parameters should be removed"
+     * But in this case extending class may need to use the colony parameter
      * @param colony The colony that owns the Work Order
      * @return True if the WorkOrder is still valid, or False if it should be deleted
      */
+    @SuppressWarnings("squid:S1172")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1172' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)